floyd/constants.py

```diff
@@ -52,6 +52,8 @@
 }
 
 DEFAULT_DOCKER_IMAGE = "floydhub/tensorflow:latest-py3"
+DEFAULT_ENV = "keras"
+DEFAULT_ARCH = "cpu"
```

Comment:
It would be better if we use "default" as the values for both of this here and set the appropriate docker image on the server. It would be easier to switch this from the server in the future.